test/generate-vdm.spec.ts

```diff
@@ -0,0 +1,172 @@
+/*!
+ * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+const confirm = jest.fn().mockResolvedValue(true);
+jest.mock('cli-ux', () => {
```

Comment:
As far as I can tell `cli.confirm()` is never used in the generate command. And the spy is also not used in the test. You should probably remove line 5 - 16.

Reply:
ah, thoughtless copy paste from the other tests